cmake/singularity-eos/ports-of-call.cmake

```diff
@@ -0,0 +1,13 @@
+macro(singularity_import_ports_of_call)
```

Comment:
Portability strategy: is it set by the host code in submodule mode and at install time for ports-of-call otherwise?

Reply:
It's set if Kokkos is enabled in either case

```cmake
if(SINGULARITY_USE_KOKKOS)
  singularity_enable_kokkos(singularity-eos)
# ...
macro(singularity_enable_kokkos target)

  target_link_libraries(${target} PUBLIC Kokkos::kokkos)

  target_compile_definitions(${target} PUBLIC PORTABILITY_STRATEGY_KOKKOS)
```
if a host code configures `singularity-eos` to use `Kokkos`, but also tries to set `PORTABILITY_STRAT_NOTKOKKOS` then there will be problems (depending on when set).